### ISSUE TITLE
Market buy, never returns price?

### ISSUE THREAD

**Issue by springjben** [jaggedsoft/node-binance-api#174]
Limit Sell response { symbol: 'STRATBTC',
  orderId: 11271474,
  clientOrderId: 'tWUdmOPSdq4bO7LltGx9IF',
  transactTime: 1522796963751,
  price: '0.00000000',
  origQty: '1.97000000',
  executedQty: '1.97000000',
  status: 'FILLED',
  timeInForce: 'GTC',
  type: 'MARKET',
  side: 'SELL' }

The 'price' field always returns 0. I want to know the estimate price it was executed at. Any ideas?

**Comment by jaggedsoft** [comment]
Yep, for anyone else interested: use `newOrderRespType` .. returns an array of fill orders 👍

**Comment by jaggedsoft** [comment]
[newOrderRespType example](https://github.com/jaggedsoft/node-binance-api/blob/master/examples/advanced.md#neworderresptype-example-when-placing-orders)
